_faq/general/create-issue.md

```diff
@@ -6,6 +6,6 @@ order: 2
 
 Great! Please file an 
 [issue](https://github.com/GoogleCloudPlatform/forseti-security/issues) 
-if it doesn't already exist. If you'd like to address the issue, let us know on 
+if it doesn't already exist. If you'd like to discuss the issue, let us know on 
```

Comment:
@Tabrarian updated several get-help sections. Please work with her to make sure this doesn't conflict or even better use the includes she created.

Reply:
Ack.

Reply:
Are you planning to make any changes here?

Reply:
I think that Michael meant you should tell me what changes to make here.  :)

Reply:
I thought the idea was to use the include? https://github.com/GoogleCloudPlatform/forseti-security/blob/2.0-forsetisecurity.org-dev/_includes/docs/latest/get_help_bug.md if not, then re-use relevant text :)

Reply:
done, ptal

Reply:
This is fine for me if Michael's also okay with it since we aren't suggesting that users should search for duplicate issues, etc.